src/Altinn.AccessManagement.Persistance/DelegationMetadataRepository.cs

```diff
@@ -252,12 +254,13 @@ private static DelegationChange GetDelegationChange(NpgsqlDataReader reader)
                 OfferedByPartyId = reader.GetFieldValue<int>("offeredbypartyid"),
                 CoveredByPartyId = reader.GetFieldValue<int?>("coveredbypartyid"),
                 CoveredByUserId = reader.GetFieldValue<int?>("coveredbyuserid"),
-                PerformedByUserId = reader.GetFieldValue<int>("performedbyuserid"),
+                PerformedByUserId = reader.IsDBNull("performedbyuserid") ? null : reader.GetFieldValue<int>("performedbyuserid"),
                 BlobStoragePolicyPath = reader.GetFieldValue<string>("blobstoragepolicypath"),
                 BlobStorageVersionId = reader.GetFieldValue<string>("blobstorageversionid"),
                 Created = reader.IsDBNull("created") ? DateTime.MinValue : reader.GetFieldValue<DateTime>("created"),
```

Comment:
not you change but I notice there already is a IsDBNull("created") which shouldn't be possible right?

Reply:
Agree Created should always have value. before and after this change.